tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Grpc/GrpcDotNetTests.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="GrpcDotNetTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.IAST.GrpcDotNet;
+
+[UsesVerify]
+public class GrpcDotNetTests : TestHelper
+{
+    public GrpcDotNetTests(ITestOutputHelper output)
+        : base("GrpcDotNet", output)
+    {
+        SetServiceVersion("1.0.0");
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.Enabled, "1");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.RedactionEnabled, "1");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.TelemetryVerbosity, "Off");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.VulnerabilitiesPerRequest, "200");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.RequestSampling, "100");
+        SetEnvironmentVariable(ConfigurationKeys.Iast.IsIastDeduplicationEnabled, "1");
+
+        SetEnvironmentVariable("IAST_GRPC_SOURCE_TEST", "1");
+    }
+
+    [SkippableFact]
+    [Trait("Category", "EndToEnd")]
+    [Trait("RunOnWindows", "True")]
+    public async Task SubmitsTraces()
+    {
+        GuardAlpine();
+        GuardLinux();
```

Comment:
We already have helpers for this 🙂 
```suggestion
        SkipOn.Platform(SkipOn.PlatformValue.Linux);
```

Unfortunately, we don't have helpers for alpine 🙁

Reply:
Oh great thanks!
Applied in daf703e